action.php

```diff
@@ -107,28 +105,40 @@ public function tplEditForm(Doku_Event &$event) {
             $form->addTagOpen('div')->addClass('form-group');
             switch ($field) {
                 case 'task':
-                    $form->addTextarea('problem[task]', $this->getLang($field))->attrs(['class' => 'form-control'])
+                    $form->addTextarea('problem[task]', $this->getLang($field))->attrs(['class' => 'form-control', 'rows' => 5])
                         ->val($problem->getTask());
                     break;
                 case 'figures':
-                    $form->addFieldsetOpen('figures');
+                    $form->addFieldsetOpen($this->getLang('figures'));
                     $figures = $problem->getFigures();
+                    $form->addTagOpen('table')->addClass('figures table table-bordered">');
+                    $form->addHTML('<tbody><tr><td>Cesta</td><td>Popisek</td><td></td></tr></tbody>');
                     if (count($figures)) {
                         foreach ($figures as $key => $figure) {
-                            $form->addTextInput('problem[figures][' . $key . '][path]', 'figures_path')
-                                ->val($figure['path']);
-                            $form->addTextInput('problem[figures][' . $key . '][caption]', 'figure_caption')
-                                ->val($figure['caption']);
+                            $form->addTagOpen('tr');
+
+                            $form->addTagOpen('td');
+                            $form->addTextInput('')->val($figure['path']);
+                            $form->addTagClose('td');
+                            $form->addTagOpen('td');
+                            $form->addTextInput('')->val($figure['caption']);
+                            $form->addTagClose('td');
+
+                            $form->addHTML('<td><button>X</button></td>');
+
+                            $form->addTagClose('tr');
                         }
-                    } else {
-                        $form->addHTML('<span class="badge badge-warning">No figures</span>');
                     }
+                    $form->addTagClose('table');
+
+                    $form->addHTML('<button class="btn btn-success btn-sm" onclick="figureManager.add();" type="button">Přidat</button>');
 
                     $form->addFieldsetClose();
                     break;
                 case 'name':
                     $form->addTextInput('problem[name]', $this->getLang($field))
                         ->attrs(['class' => 'form-control'])->val($problem->getName());
+                    $form->addHTML('<small class="form-text">Jméno úlohy bude potřeba opravit i ve FKSDB.</small>');
```

Comment:
Budeme to prekladať?

Reply:
Není třeba, jsou to interní věci, mohou zůstat v češtině. Minimálně přemístím do langu.

Reply:
:+1:  skľduom to nechajme len v češtine to bol len nápad.